.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,42 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/python
+{
+	"name": "pybravia",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"image": "mcr.microsoft.com/devcontainers/python:0-3.10-bullseye",
+
+	// Features to add to the dev container. More info: https://containers.dev/features.
+	"features": {
+	},
+
+	// Configure tool-specific properties.
+	"customizations": {
+		// Configure properties specific to VS Code.
+		"vscode": {
+			"settings": {
+				"python.pythonPath": "/usr/local/bin/python",
+				"python.languageServer": "Pylance",
+				"python.linting.enabled": true,
+				"python.linting.pylintEnabled": true,
+				"python.formatting.autopep8Path": "/usr/local/py-utils/bin/autopep8",
+				"python.formatting.blackPath": "/usr/local/py-utils/bin/black",
+				"python.formatting.yapfPath": "/usr/local/py-utils/bin/yapf",
+				"python.linting.banditPath": "/usr/local/py-utils/bin/bandit",
+				"python.linting.flake8Path": "/usr/local/py-utils/bin/flake8",
+				"python.linting.mypyPath": "/usr/local/py-utils/bin/mypy",
+				"python.linting.pycodestylePath": "/usr/local/py-utils/bin/pycodestyle",
+				"python.linting.pydocstylePath": "/usr/local/py-utils/bin/pydocstyle",
+				"python.linting.pylintPath": "/usr/local/py-utils/bin/pylint"
```

Comment:
Hello! Sorry again, I missed that.

I think we should match these linters to the project [pre-commit config](https://github.com/Drafteed/pybravia/blob/master/.pre-commit-config.yaml) and [Github Workflow](https://github.com/Drafteed/pybravia/blob/555c47a1dacd333ba0076ab18257adc4175b6c96/.github/workflows/ci.yml#L19-L54C1). 
E.g., we don't use bandit, yap, and pycodestyle linters.

Reply:
[I think cut out the non-relevant ones](https://github.com/Drafteed/pybravia/pull/24/commits/89e83f99748a0a0ab1fdf32ded668e8879a7a129). I think [this was pretty much copy-paste](https://github.com/home-assistant/core/blob/dev/.devcontainer/devcontainer.json#L18-L27) from [Home Assistant after this discussion](https://github.com/Drafteed/pybravia/pull/22#issuecomment-1453453335).

I left pylint as that's more of an IDE-focused function I believe; helps VS Code make suggestions.